src/utils/settings/index.js

```diff
@@ -127,7 +132,7 @@ const proto = {
   /**
    * Returns a settings clone with the key and traffic type (if provided) overriden.
    * @param {SplitKey} key
-   * @param {string} [trafficType]
```

Comment:
Just FYI the `[paramName]` is a way to reflect that param is nullable/optional. 
Not too important internally but it is for externally facing functions, mostly for TS declarations, as the comment will appear for intellisense.